media/css/l10n/ga-IE/intl.css

```diff
@@ -1,5 +1,5 @@
-/* Bug 854756 */
+/* Bug 1174423 */
 
 * {
-  text-transform: none !important;
```

Comment:
Let's keep this, we didn't receive any explicit request to remove it, and I don't know how bad it fails across browsers and versions.